man/chwn.1

```diff
@@ -0,0 +1,21 @@
+'\" e
+.Dd September 26, 2016
+.Dt CHWN 1
+.Os wmutils
+.Sh NAME
+.Nm chwn
+.Nd rename a window
+.Sh SYNOPSIS
+.Nm chwn
+.Ar name wid
+.Sh DESCRIPTION
+.Nm chwn
+renames
```

Comment:
I think it would be best to say it "sets the `WM_NAME` atom to *name* instead. Because there are like 100 specifications giving 200 ways to set the title for a window...